packages/node_modules/@webex/plugin-meetings/README.md

```diff
@@ -0,0 +1,43 @@
+# @webex/plugin-meetings
+
+[![standard-readme compliant](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
+
+> Logging plugin for the Cisco Webex JS SDK.
+
+- [Install](#install)
+- [Usage](#usage)
+- [Development](#development)
+- [Contribute](#contribute)
+- [Maintainers](#maintainers)
+- [License](#license)
+
+
+## Install
+
+```bash
+npm install --save @webex/plugin-meetings
+```
+
+## Usage
+
+This is a plugin for the Cisco Webex JS SDK . Please see our [developer portal](https://developer.webex.com/sdks-and-widgets.html) and the [API docs](https://webex.github.io/spark-js-sdk/api/) for full details.
+
+## Development
+
+To use `webpack-dev-server` to load this package, run `npm run serve:package -- --env.package @webex/plugin-meetings` from the root of this repo.
```

Comment:
do you think it will work on widows machine ?

Reply:
The syntax may need to change, but it should. There may be a need to install the linux subsystem to do this.